.github/workflows/test.yml

```diff
@@ -63,3 +63,55 @@ jobs:
 
       - name: Run unit tests
         run: composer phpunit
+
+  coverage:
+    needs: test
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [ '7.4' ]
+        experimental: [ false ]
```

Comment:
As the intention is to only run the code coverage on one PHP version, you may as well remove the whole `matrix` and just add `'7.4'` directly to `setup-php`.